FF149 ExprFeat/Relnote: CSS text-box and longhand props (#43468)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

- `layout.css.text-decoration-trim.enabled`
  - : Set to `true` to enable.

### `text-box`, `text-box-edge`, `text-box-trim`

The CSS {{cssxref("text-box")}} shorthand property, or the corresponding {{cssxref("text-box-trim")}} and {{cssxref("text-box-edge")}} longhand properties, allow you to trim space from the block-start and block-end edges of a text element's block container, aligning the container edges with the actual ink bounds of the text rather than the abstract box edges defined by font metrics.

> This makes it possible to achieve consistent visual spacing around text regardless of which font is used.

Reviewer: ```suggestion
This makes it possible to achieve consistent visual spacing around text regardless of the font used.
```
